<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>741</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 30px;
border: 2px solid gold;
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 gold, stop: 0.1 rgb(124, 0, 0), stop: 1.0 rgb(124, 0, 0));</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>721</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(180, 0, 0);
border-radius: 30px;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <widget class="QLineEdit" name="txtKODEBUKU">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
color: rgb(0, 0, 0);
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent;</string>
      </property>
      <property name="text">
       <string>ID Kategori</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtJudul">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent;</string>
      </property>
      <property name="text">
       <string>Judul</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCari">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>cari</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtIDKATEGORI">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
color: rgb(0, 0, 0);
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent;</string>
      </property>
      <property name="text">
       <string>Kode Buku</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent;</string>
      </property>
      <property name="text">
       <string>Tahun</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClear">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPENGARANG">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPENERBIT">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTAHUN">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent;</string>
      </property>
      <property name="text">
       <string>Pengarang</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSimpan">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Simpan</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent;</string>
      </property>
      <property name="text">
       <string>Penerbit</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(&quot;ui/binkaibulet.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>101</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(&quot;ui/aziz.jpg&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;
border: 2px solid transparent;</string>
      </property>
      <property name="text">
       <string>AZIZ MAULANA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHapus">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;
border: 2px solid transparent;</string>
      </property>
      <property name="text">
       <string>200511084</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;
border: 2px solid transparent;</string>
      </property>
      <property name="text">
       <string>R4</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="gridBuku">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>741</width>
       <height>192</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
gridline-color: rgb(124, 0, 0);</string>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>7</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent;</string>
     </property>
     <property name="text">
      <string>Kategori Dari :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNamaKategori">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border: 2px solid transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>0</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent;
background: transparent;</string>
     </property>
     <property name="text">
      <string># Data Buku #</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid transparent;
background: transparent;
color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 gold, stop:0.497326 goldenrod, stop:1 red);</string>
    </property>
    <property name="text">
     <string>CIREBON</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_16">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-image: url(&quot;ui/um1.png&quot;);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid transparent;
background: transparent;
color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 gold, stop:0.497326 goldenrod, stop:1 red);</string>
    </property>
    <property name="text">
     <string>UNIVERSITAS MUHAMMADIYAH</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-800</y>
      <width>1721</width>
      <height>1411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 75px;
border-image: url(&quot;ui/rumput.png&quot;) 0 0 0 0 stetch stretch;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>label_42</zorder>
   <zorder>frame_16</zorder>
   <zorder>label_43</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
